waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/KeyController.kt

```diff
@@ -181,6 +181,40 @@ fun Application.keys() = walletRoute {
                 }
 
         }
+        post("verify", {
+            summary = "Verify a signature with a specific key"
+            request {
+                queryParameter<String>("JWK") {
```

Comment:
this also works
but why not provide it as part of the body payload?
the body would look like:
```json
{
    "public-key":
    {
        "kty": "OKP",
        "crv": "Ed25519",
        "kid": "X2oQ9lmT7j0IeMEF1aiXQ8va7lu9PNSe2AT9orxYyHg",
        "x": "7kAIEmWZkFfdwCyWQwWw3zdfGaHwl3FcYdLDYm8Lecs"
    },
    "signature": "<signature>"
}
```
and then you'd need to define a data-transfer-object, similar to [SignRequest](https://github.com/walt-id/waltid-identity/blob/main/waltid-services/waltid-issuer-api/src/main/kotlin/id/walt/issuer/issuance/SignRequests.kt)
and do the validations with it